### PR TITLE
PIPE2D-436: Investigate wavelength solution quality

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

File: pfs/lineLists/ArCdHgKrNeXe.txt [hunk 1]
```diff
@@ -281,7 +282,7 @@
  612.496        3            HgI   0
  623.608       50            HgI   0
  670.35         2            HgI   0
- 671.819      600            HgI   3
+ 671.819      600            HgI   0
```
Comment (hassanxp) [inline pfs/lineLists/ArCdHgKrNeXe.txt]: So this line has been promoted from unreliable to OK? Why?

File: pfs/lineLists/ArCdHgKrNeXe.txt [hunk 2]
```diff
@@ -296,7 +297,7 @@
  687.26         4            HgI   0
  688.23         4            HgI   0
  689.0465       0            HgI   0
- 690.937     1000            HgI   3
+ 690.937     1000            HgI   0
```
Comment (hassanxp) [inline pfs/lineLists/ArCdHgKrNeXe.txt]: Ditto with comment above

File: pfs/lineLists/ArCdHgKrNeXe.txt (hunk 1 above)
Comment (PaulPrice) [reply pfs/lineLists/ArCdHgKrNeXe.txt]: The commit message explains:
> The previous line list vetting I did used sims data, which I
> believe only used Neon, not Hg/Ar/Kr. Some lines that were
> flagged are good lines, and useful for calibrating HgAr arcs.

Comment (hassanxp) [reply pfs/lineLists/ArCdHgKrNeXe.txt]: Ok thanks.

File: pfs/lineLists/ArCdHgKrNeXe.txt (hunk 2 above)
Comment (hassanxp) [reply pfs/lineLists/ArCdHgKrNeXe.txt]: Ok understood with the comment for L284.

**Pull request merged**